index.bs

```diff
@@ -156,6 +196,27 @@ In addition to the above spatial variations of the <a>magnetic field</a>, time b
 like solar winds or magnetic storms, also distort the magnetosphere or external magnetic field of the earth.
 
 
+
+Use Cases and Requirements {#usecases-and-requirements}
+===================
+
+Magnetometers can be used for a variety of use-cases, such as:
+
+- Metal detector web application.
+- Providing input using magnetic button for VR enclosures [[VRBUTTON]].
+- Gesture recognition based on magnetic field. Various interactions like writing, signing and playing an instrument can also be enabled using a magnet like a rod, pen or a ring [[MAGITACT]].
+  The user makes coarse gestures in the 3D space around the device using the magnet. Movement of the magnet affects the magnetic field sensed by the compass sensor integrated in the device.
+  The temporal pattern of the gesture is then used as a basis for sending different interaction commands to the mobile device. Zooming, turning pages, accepting/rejecting calls, clicking items
+  are some of the use cases.
+- In cases, where it is not possible to use [=orientation-sensor=], user may need to fuse magnetometer data with with other low-level sensor data, to calculate orientation.
+- Indoor Navigation System may use magnetometer on mobile phones [[MAGINDOORPOS]]. Specific use cases for indoor navigation may include Proximity Advertising, Way-finding in malls or airports Geo-Fencing, etc.
```

Comment:
"Indoor Navigation System", "Proximity Advertising", "Wayfinding", "Geo-Fencing" should be lowercase.